src/config/graphical-session/wayland.md

```diff
@@ -51,9 +52,8 @@ installing the `qt5-wayland` package and setting the environment variable
 applications also require installing the `kwayland` package.
 [EFL](https://wayland.freedesktop.org/efl.html)-based applications require
 setting the environment variable `ELM_DISPLAY=wl`, and can have issues without
-it, due to not supporting XWayland properly.
-[SDL](https://libsdl.org)-based applications require setting the environment
-variable `SDL_VIDEODRIVER=wayland`.
+it, due to not supporting XWayland properly. [SDL](https://libsdl.org)-based
+applications require setting the environment variable `SDL_VIDEODRIVER=wayland`.
```

Comment:
Please split into a separate commit.

Reply:
Unless this should be in a separate pull request, fixed